src/lib/components/SideNav/_styles.scss

```diff
@@ -24,7 +24,7 @@ body.web-side-nav--expanded {
   overflow: hidden;
 }
 
-web-side-nav:not(:defined) {
+body.unresolved web-side-nav {
```

Comment:
Should we do something like:

```
web-side-nav:not(:defined),
body.unresolved web-side-nav {
```

I imagine when Edgium ships we'll want to remove the web components loader stuff but we'll still want the `:not(:defined)` styles.

Reply:
Ack